sonic_platform_base/module_base.py

```diff
@@ -541,6 +616,8 @@ def get_midplane_ip(self):
         line-card and return the midplane IP-address of the line-card.
         When called from the line-card, the module will represent the
         Supervisor and return its midplane IP-address.
+        When called from the DPU, returns the midplane IP-address of the dpu-card.
+        When called from the Switch returns the midplane IP-address of Switch.
```

Comment:
@rameshraghupathy This will be called from the NPU host and for each DPU it should return the corresponding midplane ip address

Reply:
Yes, Fixed it.  Now reads
`    def get_midplane_ip(self):
        """
        Retrieves the midplane IP-address of the module in a modular chassis
        When called from the Supervisor, the module could represent the
        line-card and return the midplane IP-address of the line-card.
        When called from the line-card, the module will represent the
        Supervisor and return its midplane IP-address.
        
        When called from the SmartSwitch returns the midplane IP-address of
        the DPU module.

        Returns:
            A string, the IP-address of the module reachable over the midplane`